Sources/BraveWallet/Settings/IPFSCustomGatewayView.swift

```diff
@@ -85,8 +85,11 @@ struct IPFSCustomGatewayView: View {
       )
     }
     .onAppear {
-      if let url = ipfsAPI.nftIpfsGateway?.absoluteString {
-        self.url = url
+      // SwiftUI bug, has to wait a bit
```

Comment:
Can we put the issue number into this comment so we can understand this in the future / possibly remove when we drop iOS 15.

Reply:
added in [be85902](https://github.com/brave/brave-ios/pull/7044/commits/be85902ee0b39dd8892151ff732098e812b817a3)